backend/app/controllers/api/v1/instructor/instructors_controller.rb

```diff
@@ -13,10 +13,48 @@ def index
 
         # Find the IDs of all instructors that are associated with the same positions we are
         fellow_instructor_ids =
-            active_instructor.positions.joins(:instructors).pluck(
-                :"instructors.id"
-            )
+            active_instructor
+                .positions
+                .joins(:instructors)
+                .pluck(:"instructors.id")
 
         render_success Instructor.order(:utorid).find(fellow_instructor_ids)
     end
+
+    # POST /instructors
+    def create
+        active_user = ActiveUserService.active_user request
+
+        find_instructor
+
+        # We are only allowed to update information about ourselves, which is determined
+        # by the active user having the same utorid as the instructor to be updated.
+        unless @instructor.utorid == active_user.utorid
+            render_error(
+                message:
+                    "Insufficient permissions to update instructor with utorid='#{
+                        @instructor.utorid
+                    }'"
+            )
+        end
+
+        render_on_condition(
+            object: @instructor,
+            condition: proc { @instructor.update!(instructor_params) }
+        )
+    end
+
```

Comment:
As I understand, this code is correct. This code firstly checks if the current active user matches the instructor proposed to be changed by utorid, since the rule is that instructor can only update itself. If the check passes, the instructor is updated.